.eslintrc

```diff
@@ -43,6 +43,7 @@ env:
 
 globals:
   cy: true
+  Cypress: true
```

Comment:
I think Ans meant that we should move this whole block to the `test/.eslintrc` file.

Reply:
Yeah should look like this in `./.eslintrc`:
```
# globals:
# No globals outside of env settings.
```

And this in `./test/.eslintrc`:
```
globals:
  cy: true
  Cypress: true
```

Reply:
Thanks for clarifying and this has been done.